lib/testing-helpcenter/help_center_version_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module TestingHelpCentreOnlyoffice
+  # Help center helper for getting current hash
+  class HelpCenterVersionHelper
+    def self.fetch_uri
```

Comment:
`fetch` is then we really fetch something or download something

In this case it's just uri, maybe `def version_uri`?